apps/react-storybook/package.json

```
@@ -26,10 +27,12 @@
     "@storybook/react": "7.6.19",
     "@storybook/react-webpack5": "7.6.19",
     "@storybook/test": "7.6.19",
+    "http-server": "14.1.1",
     "prop-types": "15.8.1",
     "react": "18.0.0",
     "react-dom": "18.0.0",
     "storybook": "7.6.19",
-    "typescript": "4.9.5"
+    "typescript": "4.9.5",
+    "webpack": "5.96.1"
```

After u remove the custom webpack configuration, do we still need `webpack` dependency here?
If not -> let's remove it

```suggestion
    "typescript": "4.9.5"
```

---

Removed